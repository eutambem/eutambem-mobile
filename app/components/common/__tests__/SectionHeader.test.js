import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import SectionHeader from '../SectionHeader';

const props = {
  title: 'Test SectionHeader',
};

describe('SectionHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SectionHeader {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
