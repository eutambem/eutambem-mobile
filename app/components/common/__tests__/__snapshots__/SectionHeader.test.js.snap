// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#961687",
        "fontSize": 16,
      }
    }
  >
    TEST SECTIONHEADER
  </Text>
</View>
`;
